src/wms/src/main/java/org/geoserver/wms/capabilities/CapabilityUtil.java

```diff
@@ -188,6 +188,14 @@ public static boolean validateLegendInfo(LegendInfo legend) {
                 && legend.getWidth() > 0;
     }
 
+    /**
+     * A Utility method to populate legend url href attribute
+     *
+     * @param attrs AttributesImpl to be poplated with Legend URL href
+     * @param legendURL URL String
+     * @param XLINK_NS
```

Comment:
A check that we are about to include with the checkstyle upgrade will fail on undocumented params... either a param is there and documented, or it can be removed (won't complain for missing docs), but cannot be declared and undocumented.

Reply:
understood, will document it

Reply:
@aaime its done